draft-ietf-tls-ctls.md

```diff
@@ -595,6 +590,26 @@ This document requests that IANA open a new registry entitled "cTLS Template Key
 | finishedSize           | number       | (This document) |
 | optional               | object       | (This document) |
 
+## Reserved profiles
+
+This document requests that IANA open a new registry entitled "Well-known
+cTLS Profile IDs", on the Transport Layer Security (TLS) Parameters page,
+with the following columns:
+
+* ID value: A sequence of 1-4 octets.
+* Template: A JSON object.
+* Note: An explanation or reference.
+
+The ID value of length 1 are subject to a "Standards Action" registry
+policy. Values of length 2 are subject to an "RFC Required" policy. Other
```

Comment:
How about "2" being expert review?  Also consider making 2+3 in that category and 4+ FCFS.  Consider saying profiles starting with "-" will never be assigned by IANA, leaving that open for private use and experimentation.